### ISSUE TITLE
Warning when checking input type

### ISSUE THREAD

**Issue by snajder-r** [jhsiao999/peco#31]
Hi!

In cycle_npreg.R line 154, you have:

`if (is(Y_test) == "SingleCellExperiment") {`

This causes a warning message, since the comparison will return a vector of truth values: 

```
Warning message:
In if (is(Y_test) == "SingleCellExperiment") { :
  the condition has length > 1 and only the first element will be used
```

Did you perhaps mean to write this?

`if (is(Y_test, "SingleCellExperiment")) {`

Best,
Rene

**Comment by jhsiao999** [comment]
Hi Rene,

Thank you for catching this error! There's also been some recent minor changes to the package. I'll push these changes in a couple of days. Meanwhile, please let me know if you have any more suggestions or questions. 

Joyce

**Comment by jhsiao999** [comment]
@snajder-r Thanks for the help. The error has been fixed in peco v0.99.6.

Joyce
